lib/netsuite/rest/utilities/roles.rb

```diff
@@ -0,0 +1,85 @@
+require 'net/http'
+require 'uri'
+require 'json'
+
+module NetSuite
+  module Rest
+    module Utilities
+      class Roles
+
+        ROLES_API = "https://rest.netsuite.com/rest/roles"
+
+        def self.get(credentials)
+          new(credentials).get
+        end
```

Comment:
@davidlaprade Nice use of `self.get` to make things cleaner for the gem user. Minor nit here: using `get(*args)` and `new(*args)` would make it easier to modify the underlying arguments and self-document that the class method is just a pass-through to the child method.